Cargo.toml

```
@@ -36,6 +36,7 @@ members = [
 num-traits = { version = "0.2", default-features = false }
 num-rational = { version = "0.4", optional = true, default-features = false }
 num-bigint = { version = "0.4", optional = true, default-features = false, features = ["std"] }
+num-complex = { version = "0.4", optional = true, default-features = false }
```

I think `std` is required for `Float` types based on the skim of `num-complex` documentation that I did.

```suggestion
num-complex = { version = "0.4", optional = true, default-features = false, features = ["std"] }
```